src/System Application/App/Email/src/Account/EmailAccountWizard.Page.al

```diff
@@ -539,18 +539,16 @@ page 8886 "Email Account Wizard"
     end;
 
     local procedure LoadTopBanners()
+    var
+        GuidedExperience: Codeunit "Guided Experience";
     begin
-        if MediaResourcesStandard.Get('ASSISTEDSETUP-NOTEXT-400PX.PNG') and
-            MediaResourcesDone.Get('ASSISTEDSETUPDONE-NOTEXT-400PX.PNG') and (CurrentClientType() = ClientType::Web)
-        then
-            TopBannerVisible := MediaResourcesDone."Media Reference".HasValue();
```

Comment:
Same here, are we taking dependency from Email module to Guided Experience to load Media?

I see we have a System Module called [Image](https://github.com/microsoft/BCApps/blob/main/src/System%20Application/App/Image/src/Image.Codeunit.al), might be more suitable for this purpose